### PR TITLE
Add env to control gunicorn timeout value

### PR DESCRIPTION
**Pull request by kholdaway**

### CODE REVIEW

File: docs/source/configure.rst [hunk 1]
```diff
@@ -86,6 +86,22 @@ To view the status of the server after it is running, enter the following comman
 
   # sudo docker ps
```
Comment (abaiken) [inline docs/source/configure.rst]: Do we need to provide a way for them to set this in the install script too? That way they can just run the install script and pass it in using `-e QPC_SERVER_TIMEOUT=30` in addition to being able to manually do it starting the docker container?

Comment (kholdaway) [reply docs/source/configure.rst]: Done

File: docs/source/configure.rst [hunk 2]
```diff
@@ -86,6 +86,22 @@ To view the status of the server after it is running, enter the following comman
...
 
+Changing Other Quipucords Server Settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You may also configure the quipucords server by setting the following environment variables:
+
+  - QPC_SERVER_TIMEOUT
+    - The default http request timeout.  For example, this can be raised if you are merging a large number of jobs.
```
Comment (abaiken) [inline docs/source/configure.rst]: Maybe change jobs to reports?

File: install/install.sh [hunk 3]
```diff
@@ -51,6 +51,8 @@ usage() {
          -e QPC_DBMS_HOST=host
     * If using postgres, specify the db port (if not specified the default value is 5432):
          -e QPC_DBMS_PORT=5432
+    * Override default server timeout for HTTP requests (if not specified the default value is 120):
+         -e QPC_SERVER_TIMEOUT=120
```
Comment (abaiken) [inline install/install.sh]: We should make our instructions consistent with either QPC_SERVER_TIMEOUT or server_timeout

**Pull request merged**